example/lib/main.dart

```diff
@@ -66,6 +66,7 @@ class FlutterDeckExample extends StatelessWidget {
       // Presentation is build automatically from the list of slides.
       slides: const [
         TitleSlide(),
+        BigFactSlide(),
```

Comment:
Move the slide after `ImageSlide()` since we have a dedicated templates section starting from the `BlankSlide()`.